core/database/foxx/api/user_router.js

```diff
@@ -530,6 +578,8 @@ router.get('/token/set', function(req, res) {
     .queryParam('access', joi.string().required(), "User access token")
     .queryParam('refresh', joi.string().required(), "User refresh token")
     .queryParam('expires_in', joi.number().integer().required(), "Access token expiration timestamp")
+    .queryParam('type', joi.number().optional(), "Type of token being stored")    // TODO: currently represented as underlying integer from SDMS::AccessTokenType, do we want strings?
```

Comment:
I would make a comment where you define the Access Token Type in Foxx, class that it needs to be consistent with what is in here:

common/proto/common/SDMS.proto

We want enums. Great question. You can use the string validation with enums. 
Something like this: 
```
const AccessTokenType = { GENERIC: 1,
    GLOBUS: 2,
    GLOBUS_AUTH: 3,
    GLOBUS_TRANSFER: 4 };
```
```
'type', joi.integer().valid(...Object.values(AccessTokenType)),
```